packages/verifiable-credentials/test/config.ts

```diff
@@ -0,0 +1,50 @@
+import { AutoAccount } from '@verida/account-node';
+import { EnvironmentType, Network } from '@verida/client-ts';
+
+export const config = {
+    PRIVATE_KEY_1: "0x5dd84b6d500bcbc018cbc71b0407d694095755d91af42bd3442b2dfc96b1e0af",
+    PRIVATE_KEY_2: "0x80d3b996ec98a91536efdffbae40f5eaaf117765a587483c69195c9460165c37",
+    DID_1: 'did:vda:0xB3729982A2585544FD72c99CF3773a9c6baBD55c',
+    ADDRESS_1: '0xB3729982A2585544FD72c99CF3773a9c6baBD55c',
+    CREDENTAIL_DATA: {
```

Comment:
This is not valid credential data. `alumniOf` is not a valid attribute of that credential.

Reply:
In fact, you should create a test for this that ensures any data sent to `createCredential()` that isn't part of the schema, isn't included in the actual credential.